src/control/destination/background/mod.rs

```diff
@@ -74,6 +75,8 @@ struct DestinationCache<T: HttpService<ResponseBody = RecvBody>> {
 /// query.
 struct Config {
     namespaces: Namespaces,
+    active_queries: Arc<()>,
+    max_queries: usize,
```

Comment:
similarly, max_queries makes it sound like it could be the number of queries before which the client must be torn down, or something else. How about `concurrency_limit`?